lib/src/ReactCompareSlider.tsx

```diff
@@ -48,6 +48,7 @@ export const ReactCompareSlider = forwardRef<
       keyboardIncrement = '5%',
       style,
       transition,
+      browsingContext = window,
```

Comment:
The default value will need to be handled another way as `window` isn't a thing on the server. Not sure how best to approach this, open to suggestions.

https://github.com/nerdyman/react-compare-slider/actions/runs/7894184297/job/21561860543?pr=133#step:10:212